probe-rs/src/flashing/flasher.rs

```diff
@@ -721,12 +721,6 @@ impl<O: Operation> ActiveFlasher<'_, O> {
             }
         }
 
-        // Ensure RISC-V `ebreak` instructions enter debug mode,
-        // this is necessary for soft breakpoints to work.
-        self.core
```

Comment:
Where is this happening now?

Reply:
This is done after each reset already. https://github.com/probe-rs/probe-rs/pull/2744/files/e82b8cf2af910607a0b2a84512aec1ebc7cb80c8#diff-cc44d22620ccdaf3213479a618516ec6850085e69d35cee902de81fcfbea13bbR1724

I wanted to completely remove `debug_on_sw_breakpoint` and just enable them when initializing the debug module but that caused issues.

Reply:
Any further concerns?

Reply:
Sorry, got lost, all good.